ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -587,7 +587,7 @@ private DependencyObject ResolveView(int tag)
             return view;
         }
 
-        private IViewManager ResolveViewManager(int tag)
+        public IViewManager ResolveViewManager(int tag)
```

Comment:
While I think that makes sense from a design standpoint I think there is probably more value to having a more similar interfaces between iOS, Android, and Windows as it makes adding Windows support to existing extensions easier.